internal/bminventory/inventory.go

```diff
@@ -67,7 +67,6 @@ import (
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 	"github.com/thoas/go-funk"
-	"gopkg.in/yaml.v2"
```

Comment:
I've found it pays to always use `sigs.k8s.io/yaml` (which uses json under the hood) instead of `gopkg.in/yaml.v2`.

Reply:
On the other hand why not to use json as in all other places we just use it. Though agree that if already to use yaml better to use sigs.k8s.io/yaml in this case. 
Thanks